codemp/rd-rend2/tr_image.cpp

```diff
@@ -2569,6 +2581,155 @@ void R_UpdateSubImage( image_t *image, byte *pic, int x, int y, int width, int h
 		ri.Hunk_FreeTempMemory( resampledBuffer );
 }
 
+#define GAMMA 2.2f
```

Comment:
There are `RGBtosRGB` and `sRGBtoRGB` macros in `tr_extramath.h` which you can use instead of these